app/controllers/api/projects.php

```diff
@@ -98,6 +99,8 @@
             'keys' => null,
             'domains' => null,
             'auths' => $auths,
+            'databaseSecrets' => [\uniqid() => \bin2hex(OpenSSL::randomPseudoBytes(128))],
```

Comment:
Can we abstract this logic in the Auth library?

Reply:
getting a new secret?

Reply:
Shouldn't this be a part of OpenSSL instead of Auth library? I've put it on OpenSSL library. Do let me know WYT.